configure.ac

```diff
@@ -134,6 +134,17 @@ fi
 AC_SUBST(LIBNOTIFY_CFLAGS)
 AC_SUBST(LIBNOTIFY_LIBS)
 
+dnl -- Check for D-Bus -----------------------------------------
+AC_ARG_WITH(dbus-sys,
+        [AC_HELP_STRING([--with-dbus-sys=<dir>],
+        [where D-BUS system.d directory is])])
+if ! test -z "$with_dbus_sys" ; then
+        DBUS_SYS_DIR="$with_dbus_sys"
+else
+        DBUS_SYS_DIR='${sysconfdir}/dbus-1/system.d'
```

Comment:
Should `datadir` be here? Like in https://github.com/mate-desktop/mate-settings-daemon/pull/304.

Reply:
The new dbus default is to place it in `datadir`, however using `sysconfdir` is the safe default to make sure it works with older dbus. I am fine with either of both.

If you want to use `datadir` you should consider to change in a separate commit as it changes the current behavior.

Reply:
I think 1.10 is old enough to require it as a minimum version.

Reply:
Fine then. 👍